WorkBC Career Transition Tool WEB/careerTransitionTool/src/components/wbSelectSearchableCareer.tsx

```diff
@@ -263,7 +259,17 @@ const SelectCareer: FunctionComponent<Props> = ({
                 {!!searchText && matchedOptions?.map(option =>
                     (<Option key={option.key} value={option.value}>
                         <span style={{ whiteSpace: 'pre-line' }}>{option.label}</span>
-                    </Option>))}
+                </Option>))}
+
+            {(matchedOptions?.length > 0) && (
+                <Option className="divider" key="divider" value="divider" disabled>
+                    <span className="search-results-divider-content">
+                        {!!searchText && matchedOptions?.length === 0 && (
```

Comment:
This will never be true because of `matchedOptions.length > 0` above.. which means no matter what you type, you will never see "No match found in title". Please test this scenario.